addons/addon-base-raas/packages/base-raas-rest-api/lib/controllers/users-controller.js

```diff
@@ -74,22 +74,23 @@ async function configure(context) {
     }),
   );
 
+  // TODO: Do we want to support changing user passwords? Currently this API is only supported for internal Auth
```

Comment:
I think there's no need to support password change, if login API is gone anyways.

Reply:
deleted